api/v1alpha1/dnspolicy_types.go

```diff
@@ -69,45 +64,35 @@ type DNSPolicySpec struct {
 	// +optional
 	LoadBalancing *LoadBalancingSpec `json:"loadBalancing,omitempty"`
 
-	// +kubebuilder:validation:Enum=simple;loadbalanced
-	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="RoutingStrategy is immutable"
-	// +kubebuilder:default=loadbalanced
-	RoutingStrategy RoutingStrategy `json:"routingStrategy"`
-
 	// providerRefs is a list of references to provider secrets. Max is one but intention is to allow this to be more in the future
 	// +kubebuilder:validation:MaxItems=1
 	// +kubebuilder:validation:MinItems=1
 	ProviderRefs []dnsv1alpha1.ProviderRef `json:"providerRefs"`
 }
 
 type LoadBalancingSpec struct {
-	Weighted LoadBalancingWeighted `json:"weighted"`
-
-	Geo LoadBalancingGeo `json:"geo"`
-}
-
-// +kubebuilder:validation:Minimum=0
-type Weight int
-
-type CustomWeight struct {
-	// Label selector to match resource storing custom weight attribute values e.g. kuadrant.io/lb-attribute-custom-weight: AWS.
-	Selector *metav1.LabelSelector `json:"selector"`
-
-	// The weight value to apply when the selector matches.
-	Weight Weight `json:"weight"`
-}
-
-type LoadBalancingWeighted struct {
-	// defaultWeight is the record weight to use when no other can be determined for a dns target cluster.
+	// weight value to apply to weighted endpoints.
 	//
 	// The maximum value accepted is determined by the target dns provider, please refer to the appropriate docs below.
 	//
 	// Route53: https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/routing-policy-weighted.html
-	DefaultWeight Weight `json:"defaultWeight"`
+	// Google: https://cloud.google.com/dns/docs/overview/
+	// Azure: https://learn.microsoft.com/en-us/azure/traffic-manager/traffic-manager-routing-methods#weighted-traffic-routing-method
+	// +kubebuilder:default=120
+	Weight int `json:"weight"`
 
-	// custom list of custom weight selectors.
-	// +optional
-	Custom []*CustomWeight `json:"custom,omitempty"`
+	// geo value to apply to geo endpoints.
+	//
+	// The values accepted are determined by the target dns provider, please refer to the appropriate docs below.
+	//
+	// Route53: https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/resource-record-sets-values-geo.html
+	// Google: https://cloud.google.com/compute/docs/regions-zones
+	// Azure: https://learn.microsoft.com/en-us/azure/traffic-manager/traffic-manager-geographic-regions
+	// +kubebuilder:validation:MinLength=2
+	Geo string `json:"geo"`
+
+	// defaultGeo specifies if this is the default geo for providers that support setting a default catch all geo endpoint such as Route53.
+	DefaultGeo bool `json:"defaultGeo"`
```

Comment:
@maleck13 This could also have a default value of false if we really wanted? Minimum loadBalancing spec a user would need to specify would be:
```
loadBalancing:
  geo: EU
```
Gets you weight: 120, geo: EU, defaultGeo false in the created resources.